<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAMFIQT_GUIClass</class>
 <widget class="QMainWindow" name="CAMFIQT_GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1263</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CAMFIQT_GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonTakePic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Take Pic</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxSyncFileList">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sync</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pic Download Directory: </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDownloadDirectory">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetPicList">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>401</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pic List:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReadPicList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Read PicList</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditReadPicFrom">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>290</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>To</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditReadPicTo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Thumbnail:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelThumbnail">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Thumbnail Placeholder</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPicDownloadDirBrowse">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>371</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuration</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ISO:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxConfigIso">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto ISO:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxConfigAutoIso">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Shutter Speed:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxConfigShutterSpeed2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxConfigFNumber">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Aperture:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>White Balance:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxConfigWhiteBalance">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Exposure Meter Mode:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxConfigExposureMeterMode">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxConfigImageQuality">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Image Quality:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Exposure Program:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxConfigExposureProgram">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxPicListDoubleClick">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>171</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Double Click</string>
    </property>
    <widget class="QRadioButton" name="radioButtonDoubleClickToDownload">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Download</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonDoubleClickToDelete">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera Battery Level:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelConfigCameraBatteryLevel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>unknown</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCaptureMovie">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Live</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture Movie:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLiveMoive">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>90</y>
      <width>460</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string>Live Movie Placeholder</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1263</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="camfiqt_gui.qrc"/>
 </resources>
 <connections/>
</ui>
